src/workerd/api/sql.c++

```diff
@@ -118,6 +118,17 @@ jsg::Ref<SqlStorage::Cursor::RawIterator> SqlStorage::Cursor::raw(jsg::Lock&) {
   return jsg::alloc<RawIterator>(JSG_THIS);
 }
 
+kj::Array<jsg::V8Ref<v8::String>> SqlStorage::Cursor::getColumnNames(jsg::Lock& js) {
```

Comment:
I would add a comment here (and make sure it is reflected in the API documentation) that the array may not be a unique set and that the collection may contain duplicate names representing different distinct columns